source/app.d

```diff
@@ -109,6 +109,7 @@ shared static this()
 	settings.bindAddresses = ["127.0.0.1"];
 	settings.port = 8005;
 	settings.sessionStore = new MemorySessionStore;
+	settings.useCompressionIfPossible = true;
```

Comment:
There has been a particular fix in the ZLIB stream implementation that may have eliminated the main culprit, but using compression in has led to occasional transfer errors in the past. In order to minimize the risk we could just selectively set the compression header in the `getInfo` end point. We could also just observe whether it actually still has any issues, though, because knowing that would also be quite interesting.

It needs to be said that this setting will generally not have an effect on its own, though, because the final behavior depends on the reverse proxy facing the internet.

Reply:
> It needs to be said that this setting will generally not have an effect on its own, though, because the final behavior depends on the reverse proxy facing the internet.

Yeah, shouldn't this be handled entirely by the reverse proxy?